app/controllers/users_controller.rb

```diff
@@ -0,0 +1,29 @@
+class UsersController < ApplicationController
+
+  def index
+    @users = User.all
+    render json: @users
+  end
+
+  def create
+   @user = User.new(user_params)
+   if @user.save
+     render json: @user
+   else
+     render json: @user.errors.full_messages, status: 400
+   end
+
+  end
+
+  def users_posts
+    @posts = Post.where(user: params[:id])
```

Comment:
Huh. Didn't expect this to work, but totally does. Generally, I'd think if you're checking for a user, you'd give it a full user object not just the id.